bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -393,33 +457,38 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
                 try {
                     File f = new File(attachment);
                     FilePart fp = new FilePart(MESSAGE_KEY_ATTACHMENT, f.getName(), f);
-                    fp.setContentType("image/jpeg");
+                    String ct = contentType;
+                    if (StringUtils.isEmpty(ct)) {
+                        // ct = HttpUtil.guessContentTypeFromData(Files.readAllBytes(f.toPath()));
+                        if (StringUtils.isEmpty(ct)) {
+                            ct = DEFAULT_CONTENT_TYPE;
```

Comment:
@9037568 
i do not like the idea to read the whole file and analyse/guess the content type, this could lead to memory issues
feel free to change it in an additional PR